pkg/diagnose/firewall.go

```diff
@@ -271,14 +271,13 @@ func verifyConnectivity(localClusterInfo, remoteClusterInfo *cluster.Info, names
 	}
 
 	if options.VerboseOutput {
-		status.Success("tcpdump output from sniffer pod on Gateway node")
-		status.Success(sPod.PodOutput)
+		status.Success("tcpdump output from sniffer pod on Gateway node:\n%s", sPod.PodOutput)
 	}
 
 	if !strings.Contains(sPod.PodOutput, clientMessage) {
 		return status.Error(fmt.Errorf("the tcpdump output from the sniffer pod does not include the message"+
 			" sent from client pod. Please check that your firewall configuration allows UDP/%d traffic"+
-			" on the %q node", destPort, localEndpoint.Spec.Hostname), "Error")
+			" on the %q node. Actual pod output: \n%s", destPort, localEndpoint.Spec.Hostname, sPod.PodOutput), "")
```

Comment:
If you have a setup, can you paste the sample output in working/non-working scenario?

Reply:
I truncated the output at 500. The above text is ~430 chars so wouldn't be truncated.

Reply:
> If so I could truncate it at say 500 chars or so, ie a reasonable cushion to capture a failure like "Operation not permitted".

I'm just thinking if we really need 500 chars or can we reduce it to some lower value like 200 chars. I don't have the sample output when the "Operation not permitted" error is seen, but I guess even for such errors it should be within 200 chars.

Reply:
Anyways, lets go with 500 for now (as its logged only during errors). We can reduce it if necessary.

Reply:
OK. The "Operation not permitted" error is 77 chars so well within either limit. I fiddled around with forcing other error scenarios which could yield much more output than that. We just need enough output so we could decipher the root cause - 500 seemed a pretty good cushion. The actual tcpdump output above truncated at 500 chars doesn't seem overwhelming to me.